schema/rdl_schema_0.1.json

```diff
@@ -1088,25 +1099,31 @@
         "spatial_resolution": {
           "title": "Spatial resolution",
           "type": "number",
-          "description": "The minimum spatial separation resolvable in the resource, measured in meters.",
+          "description": "The minimum spatial separation resolvable in this resource, measured in meters.",
           "exclusiveMinimum": 0
         },
         "coordinate_system": {
           "title": "Reference coordinate system",
           "type": "string",
-          "description": "The reference co-ordinate system of the dataset version, using the [EPSG codelist](https://epsg.org).",
+          "description": "The reference co-ordinate system of this resource, using the [EPSG codelist](https://epsg.org).",
           "minLength": 1
         },
         "url": {
           "title": "Url",
           "type": "string",
-          "description": "The web address the dataset version can be downloaded from.",
+          "description": "The web address this resource can be downloaded from.",
           "format": "iri"
         },
         "temporal": {
           "title": "Temporal coverage",
           "description": "The period of time covered by this resource.",
           "$ref": "#/$defs/Period"
+        },
+        "temporal_resolution": {
```

Comment:
Is this temporal_resolution not a duplication of ln 142?

Reply:
No, because we've included the option of a temporal coverage that covers the entire dataset (ln 142) and then this one is the temporal coverage of a single resource within the dataset for use when different temporal resolutions may be present in different resources within a dataset